src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -43,6 +43,11 @@
     ServerPublicNetworkAccess
 )
 
+from azure.mgmt.monitor.models import (RetentionPolicy, LogSettings)
```

Comment:
For monitor related operation usage, please consider get_models s below and use ResourceType.MGMT_MONITOR
https://github.com/Azure/azure-cli/blob/45fae7733dc832e77fa7d4dee6542d802ea2b160/src/azure-cli/azure/cli/command_modules/network/custom.py#L1880